.travis.yml

```diff
@@ -61,7 +61,11 @@ install:
       if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
           if [[ -n "$BREW_PYTHON_PACKAGE" ]]; then
               brew update
-              brew install "$BREW_PYTHON_PACKAGE"
+              if ! brew list --versions "$BREW_PYTHON_PACKAGE" >/dev/null; then
```

Comment:
@jakubroztocil FYI this seems to fix the Travis builds that have been broken for a few days

Reply:
Figured this is better in a separate PR so moved to #705.